docs/modules/write-the-docs/pages/write-docs.adoc

```diff
@@ -0,0 +1,396 @@
+
+When creating a table, follow these guidelines:
+
+* Capitalize the initial letter of all principal words in column headings——that is, words that are not articles ("the," "an," "a"), conjunctions ("and," "but," "or"), or prepositions.
+* In most cases, do not use end punctuation for column headings, with the exception of ellipses (`...`) when the items in the column complete the phrase begun in the heading.
+* For items within a column (with the exception of the heading), use periods for whole sentences only (strings of words that include at least one verb).
+
+In most cases, the "Description" column should appear immediately following the item being described.
+
+Construct tables using the following syntax:
+
+.Example: Construct table with three columns
+[source]
+----
+[options="header", cols="1,3,1"]
+|===
+| Parameter
+| Description
+| Default
+
+| myFirstParam
+| My first long description.
+| myDefault
+
+| mySecondParam
+| My second long description.
+| myDefault
+|===
+----
+
+This is rendered as follows:
+
+[options="header", cols="1,3,1"]
+|===
+| Parameter
+| Description
+| Default
+
+| myFirstParam
+| My first long description.
+| myDefault
+
+| mySecondParam
+| My second long description.
+| myDefault
+|===
+
+For tables that are made up of more than two columns, use a separate line for each cell's content and include a blank line to separate rows, as in the code sample above.
+
+For content that has required and optional elements, use the following syntax:
+
+.Example: Construct table with required and optional elements
+[source]
+----
+[options="header", cols="1,3,1"]
+|===
+| Parameter
+| Description
+| Default
+
+3+|*Required*
+
+| myFirstParam
+| My first long description.
+| myDefault
+
+| mySecondParam
+| My second long description.
+| myDefault
+
+3+|*Optional*
+
+| myThirdParam
+| My third long description.
+| myDefault
+|===
+----
+
+Tables should have alt text associated with them; this allows screen readers to provide users with more context for the information being presented.
+Alt text should succinctly convey the table's content and function, and should not be redundant.
+If it would be redundant, omit it.
+
+== Code snippets
+
+You can include code snippets, configuration settings, or source code files in documentation.
+To enable syntax highlighting, provide the appropriate language parameter; this works for source code and configuration settings.
+
+WARNING: Use explicitly-defined code snippets as sparsely as possible.
+Code becomes obsolete very quickly, and directing to archaic practices is detrimental for users.
+
+To include code snippets, use the following syntax:
+
+.Example: Include code snippet
+====
+[source]
+----
+example code here
+----
+====
+
+If there is no suitable syntax highlighter for the language used, simply omit it, as in the example above.
+The following syntax highlighters are enabled:
```

Comment:
There's a LOT more available than what we have listed here.  I've added some additional options that are more likely to be used in our docs.
```suggestion
The following syntax highlighters are available:
```